examples/example_projects.py

```diff
@@ -45,7 +45,7 @@ def update_project_with_file_system_connector():
                                                                     included_file_names=INCLUDE_PATTERN)
     project_configuration = ProjectConfiguration(name="Test Project", project_id="test-project",
                                                  profile=ANALYSIS_PROFILE, connectors=[file_system_config],
-                                                 alias="Teamscale Python Client")
+                                                 alias="teamscale_python_client")
```

Comment:
The project alias from the old project config is now used as public ID, see `V89ProjectConfigurationMigrated#V89ProjectConfigurationMigrated()`. Thus, it may not contain spaces anymore.